generated/pom.xml

```diff
@@ -12,6 +12,22 @@
   <artifactId>generated-parent</artifactId>
   <packaging>pom</packaging>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
```

Comment:
What is the context for adding `spring-cloud-gcp-core` as a dependency here?

Reply:
Ah just realized we need it for `Credentials`,`DefaultCredentialsProvider`, etc. - got confused with something else here! Resolving.

Reply:
Great catch! I missed the fact that we have `spring-cloud-gcp` as parent pom now. Versions for both these dependencies are already specified by BOMs in [`spring-cloud-gcp/pom`](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/1c5b83dc8d039b8a71c9aa185f4bb1a328d746e9/pom.xml#L82-L96). I'll go ahead and remove the `dependencyManagement` section.

Reply:
Makes sense!